styx-scheduler-service/src/main/java/com/spotify/styx/state/handlers/ExecutionDescriptionHandler.java

```diff
@@ -107,11 +109,6 @@ private ExecutionDescription getExecDescription(WorkflowInstance workflowInstanc
 
     var workflow = workflowWithState.workflow();
 
-    var dockerImage = workflow.configuration().dockerImage().orElseThrow(
-        () -> new MissingRequiredPropertyException(
-            format("%s has no docker image, halting %s", workflowId, workflowInstance))
-    );
-
```

Comment:
is it ok to just remove the check? or should we add one more elaborate that will check for both flyteexecconf and dockerimage?

Reply:
ah i see we moved it further down